app/src/styles/ColorScheme.tsx

```diff
@@ -0,0 +1,6 @@
+/** Color Scheme */
```

Comment:
Nice job! No changes required, just something to consider:  color constants are often named as `BACKGROUND`, `FOREGROUND`, `ACCENT`, etc. to make it more flexible.

For example:
* If an application has light mode, dark mode, or other themes, calling colors "black" or "white" is restrictive. Names like "background" and "foreground" make it flexible to change the color scheme.
* If an application is re-branded to be red instead of blue, a name like "accent" or "primary_color" is more flexible for changing the brand color.

You don't need to change this since I don't think we need that flexibility here, but keep it in mind in your future work :)

Reply:
I'll keep that in mind. Thanks :>